src/@types/pkg.ts

```diff
@@ -72,6 +72,7 @@ export interface RawPkg {
   types: TsType;
   moduleTypes: ModuleType[];
   lastCrawl: string;
+  expiresAt: string;
```

Comment:
we can put it in _searchInternal to avoid exposing it too much?

Reply:
👌🏻